modules/psap-driver-toolkit-using.adoc

```diff
@@ -0,0 +1,208 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-driver-toolkit.adoc
+
+[id="using-the-driver-toolkit"]
+= Using the Driver Toolkit
+
+As an example, the Driver Toolkit can be used as the base image for building a very simple kernel module called link:https://github.com/openshift-psap/simple-kmod[simple-kmod].
+
+[id="create-simple-kmod-image"]
+== Build and run the simple-kmod driver container on a cluster
+
+.Prerequisites
+
+* An {product-title} cluster
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+First create a namespace for the example:
```

Comment:
. Create a namespace. For example:
    +
    [source,terminal]-
    ----
    $ oc new-project simple-kmod-demo
    -----